test/Array/toLocaleString.js

```diff
@@ -48,7 +48,15 @@ scenario("Array: normal");
 var o = [
     0, 1.23, NaN, true, "abc", {}, [], [0, 1, 2]
 ];
-echo(Array.prototype.toLocaleString.apply(o));
+
+// 0.toLocaleString() is supposed to be 0. However our baseline has an exception.
+// So, do not break it while supporting cross platform locale output
+var output = Array.prototype.toLocaleString.apply(o);
+if ( output == "0, 1.23, NaN, true, abc, [object Object], , 0, 1, 2" ) {
+    echo("0.00, 1.23, NaN, true, abc, [object Object], , 0.00, 1.00, 2.00");

```

Comment:
Can you clarify what's going on here? If it's supposed to be 0, is there a bug in the current Windows implementation?